src/uproot/behaviors/TBranch.py

```diff
@@ -1123,6 +1129,10 @@ def iterate(
                     ak_add_doc,
                 )
 
+                if ak_add_doc and type(out).__module__ == "awkward.highlevel":
+                    out.layout.parameters["__doc__"] = self.title
+                    out.__doc__ = self.title
+
```

Comment:
Same comment as above. (If I could highlight 3 more lines, it would include the part where `_ak_add_doc` is already applied to `out`.)